core-web/libs/block-editor/src/lib/elements/dot-bubble-menu/dot-bubble-menu.component.ts

```diff
@@ -199,10 +209,99 @@ export class DotBubbleMenuComponent {
 
         return !!image?.href;
     }
+    protected goToContentlet() {
+        // Validate selection exists before proceeding
+
+        // Get the selection
+        const selection = this.editor()?.state?.selection;
+
+        // Get the slice of the selection
+        const slice = selection?.content?.();
+
+        // Get the fragment of the slice
+        const fragment = slice?.content;
+
+        // Get the first node of the fragment
+        const selectionNode = fragment?.content?.[0];
+
+        if (!selectionNode) {
+            console.warn('Selection node is undefined, cannot navigate to contentlet');
+
+            return;
+        }
+
+        // Extract content type information from the selected node
+
+        const { data = {} } = selectionNode?.attrs || {};
+
+        const { contentType, inode } = data || {};
+
+        if (!contentType) {
+            console.warn('contentType is undefined, cannot navigate to contentlet');
+
+            return;
+        }
+
+        if (!inode) {
+            console.warn('inode is undefined, cannot navigate to contentlet');
+
+            return;
+        }
+
+        // Confirm navigation with user to prevent accidental data loss
+        if (!confirm(this.dotMessageService.get('message.contentlet.lose.unsaved.changes'))) {
+            return;
+        }
+
+        // Query content type service to determine editor capabilities and preferences
+        this.dotContentTypeService
+            .getContentType(contentType)
+            .pipe(take(1))
+            .subscribe((contentTypeInfo) => {
```

Comment:
Don't we have this state somewhere? Looks like we should have a reference of the elements that that the new editor enable by default

cc: @oidacra @nicobytes

Reply:
This request can feel slow with bad internet conection. We might need to save which contentlet have the new `edit` enable by default in `edit screen` state? Or... We can just redirect to the old editor for now...